frontend/src/components/about/Platform.vue

```diff
@@ -544,6 +400,60 @@ export default {
           ],
         },
       ],
+      citations: [
```

Comment:
That's a good idea, but I think it would require more refactoring that it looks like. If you'd like to implement this, please go ahead, or feel free to create a new issue.

Reply:
Done in: https://github.com/MetabolicAtlas/MetabolicAtlas/pull/1151/commits/11c451f3c32a8b2ca403ed0f27ae95ac6b77423b

Reply:
@mihai-sysbio if you think this refactor is ok, feel free to resolve this conversation and merge the PR.

Reply:
Resolving this conversation so you can merge the PR after reverting my commits as mentioned here: https://github.com/MetabolicAtlas/MetabolicAtlas/commit/11c451f3c32a8b2ca403ed0f27ae95ac6b77423b#r85852445